views/js/e2e/runner/navigation/linearTestNavigation.spec.js

```diff
@@ -0,0 +1,346 @@
+                cy.nextItem();
+
+                cy.get('.navi-box-list').within(() => {
+                    // visible navigation buttons
+                    cy.get(setupSelectors.testNavigation.nextItem).should('exist').and('be.visible');
+                    cy.get(setupSelectors.testNavigation.skipItem).should('exist').and('be.visible');
+
+                    // not visible navigation buttons
+                    cy.get(setupSelectors.testNavigation.previousItem).should('exist').and('not.be.visible');
+
+                    //not existing buttons
+                    cy.get(setupSelectors.testNavigation.endTest).should('not.exist');
+                    cy.get(setupSelectors.testNavigation.skipAndEndTest).should('not.exist');
+                });
+            });
+
+            it('Move to next item', () => {
+                // go to last item
+                cy.nextItem();
+                cy.nextItem();
+            });
+
+            it('Skip to next item', () => {
+                // go to last item
+                cy.nextItem();
+                cy.nextItem();
+            });
+        });
+
+        describe('Last Item', () => {
+            it('Has proper navigation buttons', function () {
+
+                // go to last item
+                cy.nextItem();
+                cy.nextItem();
+
+                cy.get('.navi-box-list').within(() => {
+                    // visible navigation buttons
+                    cy.get(setupSelectors.testNavigation.endTest).should('exist').and('be.visible');
+                    cy.get(setupSelectors.testNavigation.skipAndEndTest).should('exist').and('be.visible');
+
+                    // not visible navigation buttons
+                    cy.get(setupSelectors.testNavigation.previousItem).should('exist').and('not.be.visible');
+
+                    //not existing buttons
+                    cy.get(setupSelectors.testNavigation.skipItem).should('not.exist');
+                    cy.get(setupSelectors.testNavigation.nextItem).should('not.exist');
+                });
+            });
+
+            it('End test', () => {
+                // go to last item
+                cy.nextItem();
+                cy.nextItem();
+                cy.endTest();
+
+                //check if the test is really ended
+                cy.location().should((loc) => {
+                    expect(loc.pathname).to.eq(runnerUrls.availableDeliveriesPageUrl);
+                });
+            });
+
+            it('Skip and end test', () => {
+                // go to last item
+                cy.nextItem();
+                cy.nextItem();
+                cy.skipAndEndTest();
+
+                //check if the test is really ended
+                cy.location().should((loc) => {
+                    expect(loc.pathname).to.eq(runnerUrls.availableDeliveriesPageUrl);
+                });
+            });
+
+        });
+
+    });
+
+    describe('Non Linear Test Navigation', () => {
```

Comment:
the filename says: linearTestNavigation, in this case, please move this part to a separate file

Reply:
forgot to change file name, I will change it to navigation.js